packages/rollup-plugin-import-meta-assets/package.json

```diff
@@ -41,8 +41,10 @@
     "import-meta"
   ],
   "dependencies": {
+    "@rollup/plugin-dynamic-import-vars": "^2.0.5",
     "@rollup/pluginutils": "^5.0.2",
     "estree-walker": "^2.0.2",
+    "fast-glob": "^3.3.1",
```

Comment:
This would be the third globbing library that we have in the monorepo. Do you mind using `globby` so that it's a bit easier to manage our dependencies? 😄

Reply:
done.